MAINTAINERS.md

```diff
@@ -20,6 +20,8 @@ for these platforms.
 | HiKey Board (HiSilicon Kirin 620)	|`Linaro <op-tee@linaro.org>`|
 | HiKey960 Board (HiSilicon Kirin 960)	|`Linaro <op-tee@linaro.org>`|
 | HiSilicon D02				|`Linaro <op-tee@linaro.org>`|
+| Marvell Armada 70x0		|`Kevin Peng <kevinp@marvell.com>`|
+| Marvell Armada 80x0		|`Kevin Peng <kevinp@marvell.com>`|
```

Comment:
As Armada 80x0 is not yet supported, maybe wait for its support to declare the board.

Reply:
I'm sorry that the name "armada7080" makes confusions. It is supposed to include the two platforms, armada70x0 & armada 80x0, as they share the same codes. I shall make it clear.